src/mats_l1_processing/L1_calibration_functions.py

```diff
@@ -779,15 +779,17 @@ def desmear(image, nrskip, exptimeratio, fill=None):
     """
 
     nrow, ncol = image.shape
-    nr = nrow+nrskip
+    nr = nrow-nrskip
```

Comment:
I would consider renaming `nr` `nrow_cropped` or something. I fear these names will be confusing to someone reading this in 6 months.

Reply:
I agree some clearer variable naming or commenting would improve the readability

Reply:
mm it is not the cropped part but the number of rows of the image that are still going to be involved in the smearing. -  can think of a good name for now